source/telemetry/setup.py

```diff
@@ -1,6 +1,5 @@
 from setuptools import setup, find_packages
 
```

Comment:
I can see other places uses the `PEP8` formatting with double new line before other code is used, I think we should keep that here. Or change other places. 😄

Reply:
oh yea that is a mistake